src/lightkurve/io/eleanorlite.py

```diff
@@ -0,0 +1,112 @@
+"""Reader for GSFC-ELEANOR-LITE light curve files. 
+Details can be found at https://archive.stsci.edu/hlsp/eleanor and https://archive.stsci.edu/hlsp/gsfc-eleanor-lite
+"""
+from ..lightcurve import TessLightCurve
+from ..utils import TessQualityFlags
+from astropy import units as u
+
+from .generic import read_generic_lightcurve
+
+import numpy as np
+
+def read_eleanorlite_lightcurve(filename,
+    flux_column="CORR_FLUX",
+    quality_bitmask="default", 
+    **kwargs):
+    """Returns a `TessLightCurve` object given a light curve file from the GSFC Eleanor-lite Pipeline (https://archive.stsci.edu/hlsp/gsfc-eleanor-lite).
+
+    By default, eleanor's `CORR_FLUX` column is used to populate the `flux` values. Note that the "FLUX_ERR" column in the 
+    Eleanor FITS file is referred to the uncertainty of "RAW_FLUX", not "CORR_FLUX". 
+    Thus the uncertainty reported in the 'flux_err' column here is calculated as follows: corr_flux_err = corr_flux*raw_flux_err/raw_flux. 
+    For completeness, the original raw_flux's error is added as a "raw_flux_err" column.
+    In terms of quality flags, eleanor uses the TESS SPOC quality flags by identifying short-cadence targets that fall on each camera-CCD pairing for a given sector. However, eleanor, also adds two new quality flags -- bit 17 (decimal value 131072)) and bit 18 (decimal value 262144). 
```

Comment:
docstring style: I suggest we add a blank line before "In terms of quality flags, ", so that the quality flags text will have its own paragraph. As it is, the entire text, from `CORR_FLUX` to quality flags, are rendered as a single long paragraph that is a bit hard to read.